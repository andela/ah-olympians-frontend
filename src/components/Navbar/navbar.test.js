import React from 'react';
import { shallow } from 'enzyme';
import NavbarInstance from './Navbar';

test('Should render Navbar', () => {
  const wrapper = shallow(<NavbarInstance />);
  expect(wrapper).toMatchSnapshot();
});
