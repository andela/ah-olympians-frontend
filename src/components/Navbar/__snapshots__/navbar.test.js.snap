// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Navbar 1`] = `
<Bootstrap(undefined)
  bg="light"
  className="navbar navbar-expand-lg navbar-dark bg-dark"
  expand="lg"
  fixed="top"
>
  <Bootstrap(Image)
    alt="logo"
    responsive="true"
    src="authors.png"
    width="90"
  />
  <NavbarToggle
    aria-controls="basic-navbar-nav"
    as="button"
    label="Toggle navigation"
  />
  <NavbarCollapse
    id="basic-navbar-nav"
  >
    <Container
      as="div"
      fluid={false}
    >
      <Bootstrap(Row)>
        <Nav
          as="div"
          className="mr-auto"
          fill={false}
          justify={false}
        >
          <NavLink
            as={[Function]}
            disabled={false}
            href="#home"
          >
            Home
          </NavLink>
          <NavLink
            as={[Function]}
            disabled={false}
            href="#top"
          >
            Newest
          </NavLink>
          <NavLink
            as={[Function]}
            disabled={false}
            href="#top"
          >
            Technology
          </NavLink>
          <NavLink
            as={[Function]}
            disabled={false}
            href="#top"
          >
            Local
          </NavLink>
        </Nav>
        <Form
          as="form"
          inline={true}
        >
          <FormControl
            as="input"
            className="mr-sm-2"
            placeholder="Search"
            type="text"
          />
          <Button
            active={false}
            disabled={false}
            type="button"
            variant="outline-success"
          >
            Search
          </Button>
        </Form>
      </Bootstrap(Row)>
    </Container>
  </NavbarCollapse>
</Bootstrap(undefined)>
`;
